config/norns-crone.service

```diff
@@ -8,6 +8,7 @@ User=we
 Group=we
 LimitRTPRIO=95
 LimitMEMLOCK=infinity
+ExecStartPre=/usr/bin/jack_wait -w -t 10
```

Comment:
This should be set as an `ExecStartPost` on the `norns-jack` service, that's the one that should guarantee that JACK is running, then it doesn't need to be set in every unit that depends on JACK.

Reply:
fixed

Reply:
i could't immediate google what `jack_wait` actually does. i'm hoping this is not a 10s delay?